src/main/java/taxi/dao/jdbcimpl/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,144 @@
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        String query = "SELECT * FROM manufacturers WHERE deleted = FALSE;";
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturerList.add(createManufacturer(resultSet));
+            }
+            preparedStatement.close();
+            resultSet.close();
+        } catch (SQLException e) {
+            throw new DataProcessingException("No manufacturers found", e);
+        }
+        return manufacturerList;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String query = "UPDATE manufacturers"
+                + " SET manufacturer_name = ?, manufacturer_country = ?"
+                + " WHERE manufacturer_id = ?"
+                + " AND deleted = FALSE;";
+
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection
+                    .prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+            preparedStatement.setLong(3, manufacturer.getId());
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            while (resultSet.next()) {
+                manufacturer = createManufacturer(resultSet);
+                preparedStatement.close();
+                resultSet.close();
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can not find manufacturer with id: "
+                    + manufacturer.getId(), e);
+        }
+        return manufacturer;
+    }
+
+    private Manufacturer createManufacturer(ResultSet resultSet) throws SQLException {
+        try {
+            long manufacturerId = resultSet.getObject("manufacturer_id", Long.class);
+            String manufacturerCountry = resultSet.getObject("manufacturer_country", String.class);
+            String manufacturerName = resultSet.getObject("manufacturer_name", String.class);
+            Manufacturer manufacturer = new Manufacturer(manufacturerName, manufacturerCountry);
+            manufacturer.setId(manufacturerId);
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can not parse data"
+                    + " from manufacturer with id: "
+                    + resultSet.getObject("manufacturer_id", Long.class), e);
+        }
+    }
```

Comment:
put this private method down below all public methods

Reply:
I think it is better to keep near the methods were I use it. Also moved it closer to the related methods